resources/assets/coffee/react/profile-page/medals.coffee

```diff
@@ -32,6 +33,15 @@ ProfilePage.Medals = React.createClass
 
     _.includes @_achieved, id
 
+  _getDateAchieved: (id) ->
```

Comment:
it removes need for `_isAchieved` as well. Don't worry about the duplicated data for now.